src/charms/hash_extensions.cr

```diff
@@ -1,8 +1,30 @@
 class Hash
+  # Return the **nillable** value of a hash key
+  #
+  # This returns the value stored in the hash. Useful for getting values out of
+  # the parameter hash. Returns `nil` if the value doesn't exist:
+  #
+  # ```crystal
+  # action do
+  #   params.get(:name) # => "Karin" : (String | Nil)
+  #   params.get(:asdf) # => nil : (String | Nil)
```

Comment:
Though this example does work, the `params` method returns a `Lucky::Params` object (which also has a `get` and `get?`) method

It may be better to instantiate a new hash and use `get` on it:

```crystal
my_hash = {"name" => "Jane")
my_hash.get("name")
my_hash.get(:name)
```

It should also be worth mentioning that it converts symbols to strings. That's one of the main reasons this was added. It's like HashWithIndifferentAccess in Rails. Makes it easier to access hashes with either strings or symbols

Reply:
Ah yup, I assume they were the same. I'll put the type conversion note in there as well. Thanks!